lib/lf/refinement/CMakeLists.txt

```diff
@@ -3,10 +3,11 @@ set(sources
   hybrid2d_refinement_pattern.h hybrid2d_refinement_pattern.cc
   mesh_hierarchy.h mesh_hierarchy.cc
   refutils.h refutils.cc
+  mesh_function_transfer.h
   )
 
 lf_add_library(lf.refinement ${sources})
-target_link_libraries(lf.refinement PUBLIC Eigen3::Eigen lf.base lf.geometry lf.io)
+target_link_libraries(lf.refinement PUBLIC Eigen3::Eigen lf.base lf.geometry lf.io lf.uscalfe)
```

Comment:
I don't think we need the dependence on `lf.uscalfe` anymore?

Reply:
It's removed now.